pkg/generator/caches.go

```diff
@@ -106,7 +105,7 @@ func (caches *Caches) addTranslatedIngress(ingress *v1alpha1.Ingress, translated
 	caches.translatedIngresses[translatedIngress.name] = translatedIngress
 
 	for _, cluster := range translatedIngress.clusters {
-		caches.clusters.set(cluster, ingress.Name, ingress.Namespace)
+		caches.clusters.set(cluster, translatedIngress.name.Name, translatedIngress.name.Namespace)
```

Comment:
Should this function accept a `NamespacedName` instead of a name and a namespace? Same for the others like `caches.deleteTranslatedIngress()`.